cartography/rules/data/rules/object_storage_public.py

```diff
@@ -3,12 +3,16 @@
 from cartography.rules.spec.model import Maturity
 from cartography.rules.spec.model import Module
 from cartography.rules.spec.model import Rule
+from cartography.rules.spec.model import RuleReference
 
 # AWS Facts
 _aws_s3_public = Fact(
     id="aws_s3_public",
-    name="Internet-Accessible S3 Storage Attack Surface",
-    description=("AWS S3 buckets accessible from the internet"),
+    name="AWS S3 Buckets with Public or Anonymous Access",
```

Comment:
Nit on words: It's not that the bucket itself having access, it's more that it is internet exposed